srcpkgs/hut/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'hut'
+pkgname=hut
+version=0.1.0
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=go
+go_import_path="git.sr.ht/~emersion/hut"
+hostmakedepends="scdoc"
+short_desc="CLI tool for sr.ht"
+maintainer="Dakota Walsh <kota@nilsu.org>"
+license="AGPL-3.0-only"
+homepage="https://git.sr.ht/~emersion/hut"
+changelog="https://git.sr.ht/~emersion/hut/log"
+distfiles="https://git.sr.ht/~emersion/hut/archive/v${version}.tar.gz"
+checksum=5af8f1111f9ec1da9a818978eb1f013dfd50ad4311c79d95b0e62ad428ac1c59
+
+post_install() {
+	$GOPATH/bin/hut completion bash >hut.bash
+	$GOPATH/bin/hut completion zsh >hut.zsh
+	$GOPATH/bin/hut completion fish >hut.fish
+
+	vcompletion hut.bash bash
+	vcompletion hut.zsh zsh
+	vcompletion hut.fish fish
```

Comment:
this could be wrapped in a loop:
````sh
for $shell in bash zsh fish; do
    $GOPATH/bin/hut completion $shell > hut.$shell
    vcompletion hut.$shell $shell
done

Reply:
Will fix this and the crossbuild issues later tonight.